cpp/src/plasma/client.cc

```diff
@@ -221,18 +209,20 @@ class PlasmaClient::Impl : public std::enable_shared_from_this<PlasmaClient::Imp
 
   int get_manager_fd() const;
 
-  Status FlushReleaseHistory();
-
   bool IsInUse(const ObjectID& object_id);
 
  private:
-  /// This is a helper method for unmapping objects for which all references have
-  /// gone out of scope, either by calling Release or Abort.
+  /// Check if store_fd has already been received from the store. If yes,
+  /// return it. Otherwise, receive it from the store.
```

Comment:
Can you add a comment that this logic requires analogous logic in `ReturnFromGet` in `store.cc` to guarantee that file descriptors are sent to a client if and only if they have not been sent before.

Reply:
And probably add a comment in the relevant place in the store referring to this logic in the client.

Reply:
done